cotainr/tests/container/test_singularity_sandbox.py

```diff
@@ -158,11 +167,20 @@ def test_existing_file(self, data_cached_alpine_sif):
                 env_file.read_text().strip() == existing_shell_script + new_shell_script
             )
 
+    @pytest.mark.singularity_integration
+    def test_when_architecture_is_set(self, data_cached_alpine_sif):
+        sandbox = SingularitySandbox(base_image=data_cached_alpine_sif)
```

Comment:
Probably no need to use the actual SIF here?
```suggestion
    def test_when_architecture_is_set(self):
        sandbox = SingularitySandbox(base_image="my_base_image_6021")
```

Reply:
There is. We invoke __enter__ which requires a real container to handle subprocess-calls

Reply:
Ah yeah, I didn't notice the `_subprocess_runner` earlier in `__enter__` as it was hidden by GitHub's folding 😄